src/app/Fake.Windows.Registry/Registry.fs

```diff
@@ -0,0 +1,153 @@
+﻿/// Contains functions which allow to read and write information from/to the registry.
+///
+/// ## Sample
+/// 
+/// #### Create a subkey
+/// let subkey = "Company/MyApp"
+/// Registry.createRegistrySubKey Registry.HKEYCurrentUser subkey
```

Comment:
If you indent it like 

https://github.com/fsharp/FAKE/blob/83f79896d303c081039fd887885ca5c74c95526b/src/app/Fake.Core.ReleaseNotes/ReleaseNotes.fs#L23-L39

It will show as regular code.

Reply:
That's actually not working for me in vscode. The examples are still coming out in normal text, ReleaseNotes docs look the same way despite the indention as well. It comes out as code if I use the the md ``` marks. Ill just tab it out and recommit though